admin/sql/create_tables.sql

```diff
@@ -124,6 +124,7 @@ CREATE TABLE spotify_auth (
 CREATE TABLE external_service_oauth (
     id                      SERIAL,  -- PK
     user_id                 INTEGER NOT NULL,  -- FK to "user".id
+    external_user_id        TEXT,
```

Comment:
We also need a new SQL update script under admin/sql/updates to update the tables in production. See https://github.com/metabrainz/listenbrainz-server/blob/master/admin/sql/updates/2022-10-12-add-troi-user-setting.sql for an example.